client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/hooks/useDraftPrescriptionLines.tsx

```diff
@@ -28,13 +28,14 @@ export const useDraftPrescriptionLines = (
   item: DraftItem | null,
   date?: Date | null
 ): UseDraftPrescriptionLinesControl => {
-  const { id: invoiceId, status } = usePrescription.document.fields([
-    'id',
-    'status',
-  ]);
+  const {
+    query: { data: prescriptionData },
+  } = usePrescription();
+  const { id: invoiceId, status } = prescriptionData ?? {};
 
-  const { data: lines, isLoading: prescriptionLinesLoading } =
-    usePrescription.line.stockLines(item?.id ?? '');
+  const lines = prescriptionData?.lines.nodes.filter(
+    line => item?.id === line.item.id
+  );
```

Comment:
Again, derivable from main prescription data

Reply:
Yeah, I think the invalidation cases are all covered now, and easier to keep track of. Let's keep an eye out ;)